DNN Platform/Library/Common/Globals.cs

```diff
@@ -811,7 +811,7 @@ internal static string GetCultureCode(int TabID, bool IsSuperTab, PortalSettings
                 }
                 if (string.IsNullOrEmpty(cultureCode))
                 {
-                    cultureCode = Thread.CurrentThread.CurrentCulture.Name;
+                    cultureCode = PortalController.Instance.GetCurrentPortalSettings().DefaultLanguage;

```

Comment:
Should it be portal settings or user's personal settings? @mean2me

Reply:
portal settings is correct, IMO, since user's personal settings could be any language and, for example, in this case all Notifications will be sent according to that language, regardless of the recipient of that notification.

Reply:
I see, that makes sense. And is there a way to send them in the recipient's language?

Reply:
@tpluscode Yes, with some code refactoring. There is a PR limited to Evoq/Social at the moment.